startup_assistance/indico_startup_assistance/client/js/index.js

```diff
@@ -0,0 +1,13 @@
+/* This file is part of the CERN Indico plugins.
+ * Copyright (C) 2014 - 2018 CERN
+ *
+ * The CERN Indico plugins are free software; you can redistribute
+ * them and/or modify them under the terms of the MIT License; see
+ * the LICENSE file for more details.
+ */
+
+import setup from 'indico/modules/rb_new/setup';
+
+import overrides from './overrides';
+
+setup({...overrides});
```

Comment:
doesn't this mean we call `setup()` twice now? in burotel it's fine because we *replace* the whole room booking, but here we just want to extend it and certainly not duplicate the whole RB stuff in the plugin (otherwise the plugin needs to be rebuilt whenever something in RB changes)

Reply:
Oh, that is true! I think I shouldn't do that